content/headless/chunk/option-destination-flags.md

```diff
@@ -6,12 +6,31 @@
 
 |          |                                |
 | -------- | ------------------------------ |
-| Type:    | no-multi-line, syslog-protocol |
+| Type:    | `ignore-errors`, `no-multi-line`, `no-seqnum-all`, `seqnum-all`, `syslog-protocol`, `threaded` |
 | Default: | empty set                      |
 
+*Description:* Flags influence the behavior of the destination driver.
 
+- `ignore-errors`: <!-- FIXME -->
+- `no-multi-line`: The `no-multi-line` flag disables line-breaking in the messages: the entire message is converted to a single line.
+- `no-seqnum-all`: <!-- FIXME -->
```

Comment:
The `no-seqnum-all` flag explicitly disables new seqnum generation for forwarded messages, which is the default behavior.

---

@bazsi Do I understand this right? I feel like this *should* return 0 seqnum even for locally generated messages, but the flag does not clear the `LWO_SEQNUM` field:
https://github.com/syslog-ng/syslog-ng/blob/1f0a854d0227dd95410fbc094997d8b0b225cd41/lib/logwriter.c#L2037

Do we even need this flag? The `no-seqnum` flag (not yet released) does the full disabling, that I have mentioned. I think it is confusing to have both.

Also, why do we need the `seqnum` flag, which sets the default behavior?

I feel like there are 3 cases:
 1. local messages get a generated seqnum, forwarded messages keep their own.
 2. all messages get a generated seqnum.
 3. neither local, nor forwarded messages get a seqnum (SEQNUM = 0).

The first is the default behavior, which does not necessarily need an explicit flag.
The second is enabled by `seqnum-all`.
The third is enabled by `no-seqnum`.

I don't see the point of having any other SEQNUM related flags.